paper/paper.md

```diff
@@ -11,36 +11,57 @@ authors:
+
+For outage simulations, ERAD uses asset fragility curves [@buritica_hierarchical_2017],[@kongar_seismic_2017], [@kongar_seismic_2014], [@mo_seismic_2017], [@jeddi_multi-hazard_2022], [@baghmisheh_seismic_2021], [@williams_tsunami_2020], [@bennett_extending_2021] which are functions that relate hazard severity to survival probability for power system assets including cables, transformers, substations, roof-mounted solar panels, etc. [@rajabzadeh_improving_2022], [@fema_hazus_2020], [@farahani_earthquake_2020], [@cirone_valutazione_2013], [@sanchez-munoz_electrical_2020]. Outage scenarios are then based on Monte Carlo samples across these individual equipment survival probabilities.
+
+Grid or service resilience enhancement programs such as undergrounding, microgrids [@10.1145/3575813.3595196], and electricity backup units for critical infrastructures can also be applied, enabling users to compare metrics across options to assess their effects on equity and resilience and help identify the best approaches.
+
+# Example usage
+
+ERAD has been used as part of multiple high-impact research efforts. Specifically it was used to analyze equitable access to critical services for 8 neighborhoods in the city of Los Angeles [@la100-es-chapter12] as it transitions to a 100% renewable energy future as part of the LA100 Equity Strategies project [@la100-equity-strategy]. It has also been used to <u>*(XXX... how explicitly?)*</u> as part of North American Energy Resilience Model (NAERM) [@naerm] projects.
```

Comment:
Need a bit more specificity on how used in NAERM. Also note change to the reports for LA and NAERM cites, which seem to be more appropriate for journal articles.